framework/src/modules/dpos/types.ts

```diff
@@ -62,3 +64,14 @@ export interface RegisterTransactionAssetInput {
 export interface VoteTransactionAssetInput {
 	readonly votes: ReadonlyArray<VoteAccountAsset>;
 }
+
+export interface BlockHeaderAssetForDPOS {
+	readonly seedReveal: Buffer;
+	readonly maxHeightPreviouslyForged: number;
+	readonly maxHeightPrevoted: number;
+}
```

Comment:
remove and use `BlockHeaderAsset` from chain?

Reply:
Chain don't contain block header asset.

Reply:
https://github.com/LiskHQ/lisk-sdk/blob/af0c8df2dead9c8259ba460b1cfcf943e04cbdbb/elements/lisk-chain/src/types.ts

Reply:
mmmm i think it will added in another PR, ok for now its fine, we can clean it up later.